### ISSUE TITLE
Simplify adding java classpath entries

### ISSUE THREAD

**Issue by alex3683** [aixigo/pdfreactor-loader#2]
Currently the loader expects the `webpack.config.js` using it to add entries to the classpath, that can be retrieved calling a function of the loader module. This is overly complicated. Instead the loader module can add the classpath entry itself during module loading.

**Comment by jpommerening** [comment]
@alex3683 Good point, but I'm afraid this might break rather unpredictably.

This is about the classpath for `com.aixigo.pdfreactor_loader`, correct?

We can't guarantee that the JVM is not yet initialized when the loader is resolved. Thus, when node executes the `index.js` of the loader, the JVM might already be running (for example started by a `freemarker-loader` that ran before that).

**Comment by alex3683** [comment]
@jpommerening Good point as well ;-). Would a check using `java.isJvmCreated()` be sufficient? In the case we cannot add our classpath entry, logging would then use the fallback with the default handler of `java.util.logging.Logger`.

**Comment by jpommerening** [comment]
@alex3683 Sounds like a solid plan. Maybe handle the missing class when `try`ing to instantiate the logger? That way the user still can supply the classpath themselves.

Additionaly, we could put the classpath in a separate file, so the user does not have to figure it out themselves:
```js
java.classpath.push( require( 'pdfreactor-loader/classpath' ) );
```

**Comment by alex3683** [comment]
Nice. So in most cases one would have to configure nothing at all, and in those rare cases it's possible to get easy access to the classpath. 👍

**Comment by alex3683** [comment]
Implemented on `master` (`v0.3.0`).
